.docker/release/files/usr/local/include/application.ini

```diff
@@ -223,6 +223,7 @@ time_zone = "UTC"
 ;   filename = "log-{date:Y-m-d}.txt"
 ;   filename = "{date:Y-m-d}/{user:domain}/{user:email}_{user:uid}.log"
 ;   filename = "{user:email}-{date:Y-m-d}.txt"
+;   filename = "stderr"
```

Comment:
this file should not be part of the docker build context, it needs to be generated as part of `release.php`, or some other way so that it doesn't go out of state with `snappymail/v/0.0.0/app/libraries/RainLoop/Config/Application.php`. I'll try to find a way.

Reply:
resolved in ddc30b5295e93e287e066b946a5179516620d36d